data/en/1_0005.json

```diff
@@ -733,6 +733,7 @@
                                     "max_value": {
                                         "value": 100
                                     },
+                                    "decimal_places": 2,
                                     "validation": {
                                         "messages": {
                                             "NUMBER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
```

Comment:
does line 760: "weekly-pay-significant-changes-pay-rates-increase-percent-employees-answer" need:
validation and 2 decimal places?